assets/app/view/create_game.rb

```diff
@@ -45,10 +45,12 @@ def render_content
           }
         end
 
+        inputs << h(:br)
+        inputs << render_button('Upload button') { `document.getElementById('import_json_file').click()` }
         inputs << h(:input, {
                       attrs: { type: :file, id: :import_json_file, accept: '.json' },
                       on: { change: upload },
-                      style: { background: :unset },
+                      style: { visibility: :hidden },
```

Comment:
The element will take up space nonetheless, which likely will result in blank line and possibly even overflow on mobile.
Please test thoroughly.

Reply:
I wasn't afraid of the possible extra line, because I was already in doubt if there should be even more space :P

In any case, I moved it to the bottom to avoid that. It can't be hidden with `display: none`, because that prevents the event from triggering.

Reply:
I see no trace of a width-limit. Have you tested for overflow on mobile?

Reply:
It is just a single element on the bottom, it shouldn't be problematic.

Reply:
Sounds like "No". Test it. If it overflows (which I expect it does) fix it.

Reply:
I didn't test in all phones, no.

If some device overflows with this, it will overflow with any other file input. But feel free to propose an alternative.

Reply:
Dude …
It’s not about the file dialog. That will be handled natively and definitely not overflow.
It’s about the invisible html-element that takes up space nonetheless. It’s too wide for the page => overflow => wiggling page on mobile => no go!
Just limit the element’s width and test that the overflow is gone afterwards. No need to do this on "all phones" or any phone at all. You could just use responsive design mode (web-dev) in your preferred browser and set width to 320 px …

Reply:
Errr... Where did I write anything about the dialog? wtf...

The width will depend on the browser you're using (as you've just shown in the other PR). That's what I meant by "all phones": not all devices, but all configurations (operating systems + browsers + whatever)

> You could just use responsive design mode (web-dev) in your preferred browser and set width to 320 px
Usually I just go for the lazy option and resize the browser xD